packages/katello/katello/hostname-change.rb

```diff
@@ -151,7 +151,7 @@ def warning_message
     end
 
     def hammer_cmd(cmd, exit_codes=[0], message=nil)
-      run_cmd("hammer -u #{@options[:username].shellescape} -p #{@options[:password].shellescape} #{cmd}", exit_codes, message)
+      run_cmd("LANG=en_US.utf-8 hammer -u #{@options[:username].shellescape} -p #{@options[:password].shellescape} #{cmd}", exit_codes, message)
```

Comment:
I do think it's more common to use the uppercase encoding. This is what all the manpages use as examples:
```suggestion
      run_cmd("LANG=en_US.UTF-8 hammer -u #{@options[:username].shellescape} -p #{@options[:password].shellescape} #{cmd}", exit_codes, message)
```